docs/cli/graphql-transformer/config-params.md

```diff
@@ -103,43 +103,43 @@ Follow these two steps when you need to rotate an API Key
 }
 ```
 
-## ElasticsearchStreamingFunctionName
+## ElasticSearchStreamingFunctionName
 
 **Override the name of the AWS Lambda searchable streaming function**
 
 ```
 {
-  "ElasticsearchStreamingFunctionName": "CustomFunctionName"
+  "ElasticSearchStreamingFunctionName": "CustomFunctionName"
 }
 ```
 
-## ElasticsearchInstanceCount
+## ElasticSearchInstanceCount
 
 **Override the number of instances launched into the Elasticsearch domain created by @searchable**
 
 ```
 {
-  "ElasticsearchInstanceCount": 1
+  "ElasticSearchInstanceCount": 1
 }
 ```
 
-## ElasticsearchInstanceType
+## ElasticSearchInstanceType
 
 **Override the type of instance launched into the Elasticsearch domain created by @searchable**
 
 ```
 {
-  "ElasticsearchInstanceType": "t2.small.elasticsearch"
+  "ElasticSearchInstanceType": "t2.small.elasticsearch"
 }
 ```
 
-## ElasticsearchEBSVolumeGB
+## ElasticSearchStreamingFunctionName
```

Comment:
Why is this mentioned twice? Shouldn't this be "ElasticsearchEBSVolumeGB" - given the example below where the value is 10?

Reply:
should be "ElasticSearchEBSVolumeGB" I think.

Reply:
Fixed